include/librealsense2/dds/topics/control/control.idl

```diff
@@ -0,0 +1,24 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+module librealsense
+{
+    module dds
+    {
+        module topics
+        {
+            module raw
+            {
+                module device
+                {
+                    struct control
+                    {
+                        unsigned short id; // See "librealsense::dds::topics::control::msg_type"
+                        unsigned long size;
+                        sequence<octet> raw_data;
+                    };
```

Comment:
Possibly missing the ID of the message (sequential or other) and maybe the GUID of the sender?

Reply:
Will be discussed in the meeting

Reply:
Will be changed, as discussed, in a different PR